suspendmanager.lua

```diff
@@ -433,6 +639,11 @@ function SuspendManager:refresh()
             end
         end
 
+        -- Check for construction jobs which may be unsupported
```

Comment:
This block should be moved under the ```Internal reasons to suspend a job```.

The terminology is not very good, but everything before this check is what is already suspended by dwarf fortress, and suspendmanager (/unsuspend) will never unsuspend it, and everything after is what suspendmanager is actively suspending.

Reply:
Ok, I misunderstood what `preventBlocking` meant in this context. I think maybe that might warrant a rename due to the expanded scope of this script? perhaps something like `preventRiskyJobs` or something?

Reply:
It's actually even wider than that, the suspended jobs on smoothing designation also fall in that category, they are not risky, they would just erase a smoothing job. Maybe falling back on the broader "smart" term would be best.

This issue is spread all across the suspension script, even in the UI and the `suspend`/`unsuspend` command line options, I suggest not to do that in that pr, maybe a separate issue?